src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,79 @@
+package core.basesyntax;
+
+import core.basesyntax.dao.FruitStorageDao;
+import core.basesyntax.dao.FruitStorageDaoImpl;
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.service.DataConverterService;
+import core.basesyntax.service.FileReaderService;
+import core.basesyntax.service.FileWriterService;
+import core.basesyntax.service.ReportGenerator;
+import core.basesyntax.service.ShopService;
+import core.basesyntax.service.impl.CsvFormatReportGenerator;
+import core.basesyntax.service.impl.DataConverterServiceImpl;
+import core.basesyntax.service.impl.FileReaderServiceImpl;
+import core.basesyntax.service.impl.FileWriterServiceImpl;
+import core.basesyntax.service.impl.ShopServiceImpl;
+import core.basesyntax.service.impl.operation.BalanceOperation;
+import core.basesyntax.service.impl.operation.OperationHandler;
+import core.basesyntax.service.impl.operation.PurchaseOperation;
+import core.basesyntax.service.impl.operation.ReturnOperation;
+import core.basesyntax.service.impl.operation.SupplyOperation;
+import core.basesyntax.strategy.OperationStrategy;
+import core.basesyntax.strategy.OperationStrategyImpl;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class Main {
+    private static final FruitStorageDao FRUIT_STORAGE_DAO = new FruitStorageDaoImpl();
+
+    public static void main(String[] arg) {
+        System.out.println(Storage.fruits);
+        System.out.println("-----------------------");
+        // 1. Read the data from the input CSV file
+        FileReaderService fileReader = new FileReaderServiceImpl();
+        List<String> inputReport = fileReader.read("src/main/resources/reportToRead.csv");
+
+        inputReport.stream()
+                .forEach(System.out::println);
```

Comment:
Please remove the console output used for debugging. If you need to log information, consider using a logging framework.